building gateway on MacOSX 10.7+ can yield " group id '2119906183' is too big ( > 2097151 )" assembly error

When I built the gateway distribution on Mac OSX 10.9, I received the following error:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.5.1:attached    (default) on project gateway.distribution.base: Execution default of goal org.apache.maven.plugins:maven-assembly-plugin:2.5.1:attached failed: group id '2119906183' is too big ( > 2097151 ) -> [Help 1]
```

After some research it is a known issue with maven assembly plugin 2.5+, and resolved  via configuration change in http://jira.codehaus.org/browse/MASSEMBLY-728

--- a/base/pom.xml
+++ b/base/pom.xml
@@ -310,6 +310,7 @@
                                 <descriptor>src/main/assembly/unix-bin.xml</descriptor>
                                 <descriptor>src/main/assembly/windows-bin.xml</descriptor>
                             </descriptors>
+                            <tarLongFileMode>posix</tarLongFileMode>
                         </configuration>
                     </execution>
                 </executions>

